features/your_results_frequency_selector.feature

```diff
@@ -1,36 +1,69 @@
 Feature:
-  As an employee, I want to be able to see my results in a frequency different to how I input my salary because I want to understand how the contributions will impact my monthly or weekly take-home pay.
+  In order to understand how the contributions will impact my monthly or weekly take-home pay
+  As an employee
+  I want to be able to see my results in a frequency different to how I input my salary
 
   Background:
     Given I am on the Your Details step
+    And I am a "25" year old "female"
```

Comment:
Seeing the cucumber book again, the books recommends to create some sort of profiles on the background, so I am keen to have the step: `And I am a "25" year old "female"`. Let's move this discussion to slack.